fabric-network/test/network.js

```diff
@@ -258,161 +229,74 @@ describe('Network', () => {
 
 		it('calls close() on its channel', () => {
 			network._dispose();
-			sinon.assert.calledOnce(mockChannel.close);
+			sinon.assert.calledOnce(channel.close);
 		});
 
-		it('calls dispose() on the event hub factory', () => {
-			const spy = network.getEventHubManager().dispose;
+		it('calls dispose() on the event service factory', () => {
+			const spy = network.eventServiceManager.dispose;
 			network._dispose();
 			sinon.assert.called(spy);
 		});
 
 		it('calls unregister on its listeners', () => {
-			const mockListener = sinon.createStubInstance(BlockEventListener);
-			network.listeners.set('mockListener', mockListener);
+			const listener = sinon.createStubInstance(BlockEventListener);
+			network.listeners.set('listener', listener);
 			network._dispose();
-			sinon.assert.calledOnce(mockListener.unregister);
+			sinon.assert.calledOnce(listener.unregister);
 		});
 	});
 
-	describe('#getQueryHandler', () => {
```

Comment:
`transaction.js` still relies on the query handler being available on the Network, with the only difference being that it accesses the member directly rather than uses `Network.getQueryHandler()`. So these tests still seem completely valid and should not be removed.